Tasks/Common/codecoverage-tools/module.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you delete this file if not needed? I'm not sure whether it will cause an issue during loc handoff/handback to have an empty file.